lutris/runners/wine.py

```diff
@@ -594,6 +646,7 @@ class wine(Runner):
     reg_prefix = "HKEY_CURRENT_USER/Software/Wine"
     reg_keys = {
         "Audio": r"%s/Drivers" % reg_prefix,
+        "Graphics": r"%s/Drivers" % reg_prefix,
```

Comment:
Can we avoid setting this when using Proton? Not obvious how to do this.

Reply:
I think I see the place- about line 1000 of this file, it already special cases ``Desktop`` and ``WineDesktop`` to be skipped sometimes. You could also do ``Graphics`` there.

Reply:
I saw this too and made the same solution, hopefully it will work properly.